src/main/features/dashboard/views/contact-them.njk

```diff
@@ -0,0 +1,111 @@
+  {% set address = claim.response.defendant.address if claim.response.defendant.address else claim.claimData.defendant.address   %}
+  {% set correspondenceAddress = claim.response.defendant.correspondenceAddress if claim.response.defendant.correspondenceAddress else claim.claimData.defendant.correspondenceAddress %}
+  {% set contactPerson = claim.response.defendant.contactPerson %}
+
+{% elif user.id === claim.defendantId %}
+  {% set otherParty = 'claimant' %}
+  {% set otherPartyName = claim.claimData.claimant.name %}
+  {% set phone = claim.claimData.claimant.mobilePhone %}
+  {% set email = claim.claimantEmail %}
+  {% set address = claim.claimData.claimant.address %}
+  {% set correspondenceAddress = claim.claimData.claimant.correspondenceAddress %}
+  {% set contactPerson = claim.claimData.claimant.contactPerson %}
+
+{% endif %}
+
+{% set heading = 'Contact ' + t(otherParty) %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">
+        <span class="heading-secondary">{{ t(otherParty) | capitalize }}</span>
+        {{ otherPartyName }}
+      </h1>
+      <div class="notice">
+        {% if address %}
+          <h2 class="heading-medium">{{ t('Address') }}</h2>
+          {% if address.line1 %}
+            <span class="block">{{ address.line1 }}</span>
+          {% endif %}
+
+          {% if address.line2 %}
+            <span class="block">{{ address.line2 }}</span>
+          {% endif %}
+
+          {% if address.line3 %}
+            <span class="block">{{ address.line3 }}</span>
+          {% endif %}
+
+          {% if address.city %}
+            <span class="block">{{ address.city }}</span>
+          {% endif %}
+
+          {% if address.postcode %}
+            <span class="block">{{ address.postcode }}</span>
+          {% endif %}
+
+        {% endif %}
+      </div>
+
+      {% if correspondenceAddress %}
+        <div class="notice">
+          <h2 class="heading-medium">{{ t('Correspondence address') }}</h2>
+          {% if correspondenceAddress.line1 %}
+            <span class="block">{{ correspondenceAddress.line1 }}</span>
+          {% endif %}
+
+          {% if correspondenceAddress.line2 %}
+            <span class="block">{{ correspondenceAddress.line2 }}</span>
+          {% endif %}
+
+          {% if correspondenceAddress.line3 %}
+            <span class="block">{{ correspondenceAddress.line3 }}</span>
+          {% endif %}
+
+          {% if correspondenceAddress.city %}
+            <span class="block">{{ correspondenceAddress.city }}</span>
+          {% endif %}
+
+          {% if correspondenceAddress.postcode %}
+            <span class="block">{{ correspondenceAddress.postcode }}</span>
+          {% endif %}
+        </div>
+      {% endif %}
+
+      <div>
+        {% if phone %}
+          <span class="block">{{ t('Phone: ' ) + phone }}</span>
+        {% endif %}
+
+        {% if email %}
```

Comment:
same here. we know we have their email address. should be no need for the if statements

Reply:
actually we may not have the email address of the defendant

Reply:
Updated accordingly.